### PR TITLE
feat: add list component

### PR DESCRIPTION
**Pull request by gadgetfather**
I have created a list component please review it!

preview link: https://jwidp.csb.app/list-component/list.html

image

### CODE REVIEW

File: list-component/list.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@import url(/common.css);
+
+.list{
+    border: 1px solid black;
+    width: 12rem;
+    display: flex;
+    flex-direction: column;
+    gap: var(--spacing7);
+    /* align-items: center; */
```
Comment (shaik-saad) [inline list-component/list.css:9]: You can remove the comment if not needed.

File: list-component/list.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    padding: 1rem;
+    margin: 0;
```
Comment (shaik-saad) [inline list-component/list.css:11]: You have reset the margin in common.css file, so you can remove the margin here.

**Review by shaik-saad — CHANGES_REQUESTED** [review]
Suggested some changes, do check them out. Also in LMS spaced list is also mentioned, do try to make the spaced list too.

File: list-component/list.css (hunk 2 above)
Comment (gadgetfather) [reply list-component/list.css:11]: yea right!

File: list-component/list.css (hunk 1 above)
Comment (gadgetfather) [reply list-component/list.css:9]: i will do it

**Review by mohammed-sas — APPROVED** [review]
other than suggested by saad , your pr is good to merge

File: list-component/list.css [hunk 3]
```diff
@@ -26,4 +26,16 @@
...
     padding: 0.3rem 0.2rem;
     background-color: lightgray;
+}
+
+/* spaced */
+.list-spaced{
+    flex-direction: row;
+    align-items: center;
+    width: 30rem;
+    /* list-style: circle; */
```
Comment (shaik-saad) [inline list-component/list.css:36]: You can remove this comment.

**Review by shaik-saad — APPROVED** [review]
Great! You're good to merge now.

**Pull request merged**